src/main/java/dl/news/portal/domain/dto/UserDto.java

```diff
@@ -31,6 +31,9 @@ public UserDto(String username, String email, String password) {
         this.password = password;
     }
 
+    public UserDto() {
```

Comment:
Why do you need it?

Reply:
If there are no primary or default constructor IllegalStateException(caused by NoSuchMethodException) is thrown. It's arise when object is deserialized from json.